src/current/v23.2/molt-fetch.md

```diff
@@ -0,0 +1,413 @@
+
+{% include_cached copy-clipboard.html %}
+~~~
+--table-filter '\w*user\w*'
+~~~
+
+### Target table handling
+
+`--table-handling` defines how MOLT Fetch loads data on the CockroachDB tables that [match the selection](#schema-and-table-selection).
+
+To load the data without changing the existing data in the tables, use `'none'`:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--table-handling 'none'
+~~~
+
+To [truncate]({% link {{ page.version.version }}/truncate.md %}) tables before loading the data, use `'truncate-if-exists'`:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--table-handling 'truncate-if-exists'
+~~~
+
+To drop existing tables and create new tables before loading the data, use `'drop-on-target-and-recreate'`:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--table-handling 'drop-on-target-and-recreate'
+~~~
+
+With each option, MOLT Fetch creates a new CcokroachDB table to load the source data if one does not exist.
+
+### Fetch continuation
+
+If `molt fetch` exits with an error after being run, you can continue the process from the *continuation point* where it was interrupted.
+
+To retry all data starting from the continuation point, include `--fetch-id` and specify the process ID from the `molt fetch` output.
+
+{% include_cached copy-clipboard.html %}
+~~~
+--fetch-id d44762e5-6f70-43f8-8e15-58b4de10a007
+~~~
+
+To retry a specific table that failed, include both `--fetch-id` and `--continuation-token`. The latter flag specifies a token string that corresponds to a specific table on the source database. A continuation token is written in the `molt fetch` output for each failed table. If the fetch process encounters a subsequent error, it generates a new token for each failed table.
+
+{{site.data.alerts.callout_info}}
+This will retry only the table that corresponds to the continuation token. If the fetch process succeeds, there may still be source data that is not yet loaded onto CockroachDB.
+{{site.data.alerts.end}}
+
+{% include_cached copy-clipboard.html %}
+~~~
+--fetch-id d44762e5-6f70-43f8-8e15-58b4de10a007
+--continuation-token 011762e5-6f70-43f8-8e15-58b4de10a007
+~~~
+
+To retry all data starting from a specific file, include both `--fetch-id` and `--continuation-file-name`. The latter flag specifies the filename of an intermediate file in [cloud or local storage](#data-path). All filenames are prepended with `part_` and have the `.tar.gz` or `.csv` extension, depending on compression type (gzip by default). For example: 
+
+{% include_cached copy-clipboard.html %}
+~~~
+--fetch-id d44762e5-6f70-43f8-8e15-58b4de10a007
+--continuation-file-name part_00000003.tar.gz
+~~~
+
+{{site.data.alerts.callout_info}}
+Continuation is not possible when using [direct copy mode](#direct-copy).
+{{site.data.alerts.end}}
+
+### CDC cursor
+
+If the fetch process succeeds, a change data capture (CDC) cursor is written to the output as `cdc_cursor`. For example:
```

Comment:
We output the cursor at the beginning as well. So it depends on when you are setting up CDC. If you use import, most likely after since we take your table offline. Copy you could set it up right away.